lambdaFunctions/phsDailyDataProcessing/src/s3Utils.js

```diff
@@ -0,0 +1,274 @@
+import fetch from "node-fetch";
+const AWS = require("aws-sdk");
+const s3 = new AWS.S3();
+
+const OBJECT_FOLDER = "data/";
+
+const NHS_SCOT_URL_PREFIX = "https://www.opendata.nhs.scot/dataset/";
+const NHS_SCOT_DAILY_URL_PREFIX =
+  NHS_SCOT_URL_PREFIX + "b318bddf-a4dc-4262-971f-0ba329e09b87/resource/";
+
+const MAIN_DATASETS = [
+  {
+    url:
+      NHS_SCOT_DAILY_URL_PREFIX +
+      "427f9a25-db22-4014-a3bc-893b68243055/download",
+    objectKey: OBJECT_FOLDER + "dailyCouncilAreas.csv",
+  },
+  {
+    url:
+      NHS_SCOT_DAILY_URL_PREFIX +
+      "2dd8534b-0a6f-4744-9253-9565d62f96c2/download",
+    objectKey: OBJECT_FOLDER + "dailyHealthBoards.csv",
+  },
+  {
+    url:
+      NHS_SCOT_DAILY_URL_PREFIX +
+      "e8454cf0-1152-4bcb-b9da-4343f625dfef/download",
+    objectKey: OBJECT_FOLDER + "currentTotalsCouncilAreas.csv",
+  },
+  {
+    url:
+      NHS_SCOT_DAILY_URL_PREFIX +
+      "7fad90e5-6f19-455b-bc07-694a22f8d5dc/download",
+    objectKey: OBJECT_FOLDER + "currentTotalsHealthBoards.csv",
+  },
+];
+
+const ADDITIONAL_DATASETS = [
+  {
+    url:
+      NHS_SCOT_DAILY_URL_PREFIX +
+      "3349540e-dc63-4d6d-a78b-00387b9aca50/download",
+    objectKey: OBJECT_FOLDER + "testsCouncilAreas.csv",
+  },
+  {
+    url:
+      NHS_SCOT_DAILY_URL_PREFIX +
+      "8da654cd-293b-4286-96a4-b3ece86225f0/download",
+    objectKey: OBJECT_FOLDER + "testsHealthBoards.csv",
+  },
+  {
+    url:
+      NHS_SCOT_DAILY_URL_PREFIX +
+      "287fc645-4352-4477-9c8c-55bc054b7e76/download",
+    objectKey: OBJECT_FOLDER + "dailyAndCumulativeTests.csv",
+  },
+  {
+    url:
+      NHS_SCOT_DAILY_URL_PREFIX +
+      "8906de12-f413-4b3f-95a0-11ed15e61773/download",
+    objectKey: OBJECT_FOLDER + "weeklyTrendsByNeighbourhood.csv",
+  },
+  {
+    url:
+      NHS_SCOT_DAILY_URL_PREFIX +
+      "9393bd66-5012-4f01-9bc5-e7a10accacf4/download",
+    objectKey: OBJECT_FOLDER + "dailyCaseTrendsByAgeAndSex.csv",
+  },
+  {
+    url:
+      NHS_SCOT_DAILY_URL_PREFIX +
+      "a38a4c21-7c75-4ecd-a511-3f83e0e8f0c3/download",
+    objectKey: OBJECT_FOLDER + "dailyCaseTrendsByDeprivation.csv",
+  },
+  {
+    url:
+      NHS_SCOT_URL_PREFIX +
+      "d67b13ef-73a4-482d-b5df-d39d777540fd/resource/5acbccb1-e9d6-4ab2-a7ac-f3e4d378e7ec/download",
+    objectKey: OBJECT_FOLDER + "hospitalOnset.csv",
+  },
+];
+
+// const BUCKET_NAME = "dashboard.aws.scottishtecharmy.org";
+const BUCKET_NAME = "dashcoviddata";
```

Comment:
Temporary bucket name till we're done merging - don't want both lambda implementations hitting the same files...